doc/user/connect/cli.md

```diff
@@ -20,9 +21,17 @@ Detail | Info
 
 Tool | Description | Install
 -----|-------------|--------
-`mzcli` | Materialize adaptation of `pgcli` | [GitHub](https://github.com/MaterializeInc/mzcli#quick-start)
-`psql` | Vanilla PostgreSQL CLI tool. Useful for executing queries from scripts using `-c`. | `libpq` or `postgresql-client`
-`pgcli` | CLI tool with auto-completion & syntax highlighting | [`pgcli` documentation](https://www.pgcli.com/install)
+`mzcli` | Materialize-specific CLI | [GitHub](https://github.com/MaterializeInc/mzcli#quick-start)
+`psql` | Vanilla PostgreSQL CLI tool | `postgresql` or `postgresql-client`
+
+{{< warning >}}
+Not all features of `psql` are supported by Materialize.
+{{< /warning >}}
+
+Other tools built for PostgreSQL can often be made to work with Materialize with
+minor modifications, but are unlikely to work out of the box.
+[File a GitHub issue](https://github.com/MaterializeInc/materialize/issues/new?labels=C-feature&template=feature.md)
+if there is a PostgreSQL tool you would like us to consider supporting.
```

Comment:
Maybe emphasize that this is more of "we're not finished" issue?

>While it is a goal for tools built for PostgreSQL to work out of the box, they may require minor or major modifications to Materialize. Please [File a GitHub issue](https://github.com/MaterializeInc/materialize/issues/new?labels=C-feature&template=feature.md) if there is a PostgreSQL tool you would like us to prioritize supporting.

Reply:
Is it a goal for all tools built for PostgreSQL to work out of the box? I'm not personally ready to make that commitment!